src/components/landing/EventsListItem/EventListCard/EventListCard.js

```diff
@@ -3,9 +3,9 @@ import { NavLink } from "react-router-dom";
 import PropTypes from "prop-types";
 import PhotoService from "../../../../services/PhotoService";
 import eventDefaultImage from "../../../../constants/eventDefaultImage";
-import parseDate from "../../../helpers/parseDate";
 import IconsEventCard from "../../EventCard/IconsEventCard/IconsEventCard";
 import "./EventListCard.scss";
```

Comment:
Same for style import. I know that it's out of scope of your changes, but please fix it. IT's very small

Reply:
Done